### PR TITLE
Bugfix/fp 1542 do not use a2cps url

### PR DESCRIPTION
**Pull request by wesleyboar**
## Overview

Use local images for Data Transfer guides.

## Related

- [FP-1542](https://jira.tacc.utexas.edu/browse/FP-1542)
- requires https://github.com/TACC/Core-Styles/pull/25

## Changes

- **[fix] add and use images**
- [enhancement] add extra example images I found
- [fix] tweak image max-width

## Testing

Confirm images:

- are **not** loaded from `a2cps-staging`
- are **not** wider than the image max width*
- are **not** excessively large
- are **not** rendered larger than intrinsic dimensions†

<sub>* The default guide max image width is 800px, but Data Transfer page sets 670px max image width.</sub>
<sub>† Some of the images are just bad quality.</sub>

### Remote

- https://dev.cep.tacc.utexas.edu/guides/data-transfer-guide/
- https://dev.cep.tacc.utexas.edu/guides/globus-data-transfer-guide/

<sub>[build job](https://jenkins01.tacc.utexas.edu/job/Core_CMS_Build/646/) & [deploy job](https://jenkins01.tacc.utexas.edu/job/Core_Portal_Deploy/1472/)</sub>

### Local

1. Be able to [Test CMS Changes](https://github.com/TACC/Core-CMS/wiki/Test-CMS-Changes).
    - remember to `collectstatic` (to provide the local images to Django)
3. Add page/s set to use these templates:
    - "Guide: Globus Data Transfer"
    - "Guide: Data Transfer"

## Screenshots

| Data Transfer | Globus Data Transfer |
| - | - |
| FP-1542-data-transfer | FP-1542-globus-data-transfer |

## Notes

<details><summary>✓ To Do</summary>

- [x] [Generice-ize images.](https://github.com/TACC/Core-CMS/pull/493#pullrequestreview-983879502) b16bb9f
- [x] Create ticket for more changes. [FP-1676](https://jira.tacc.utexas.edu/browse/FP-1676)

</details>

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
       "version": "3.7.0",
       "license": "MIT",
       "devDependencies": {
-        "@tacc/core-styles": "git+https://git@github.com/TACC/Core-Styles.git",
+        "@tacc/core-styles": "git+https://git@github.com/TACC/Core-Styles.git#bugfix/fp-1542-figure-image-margin",
```
Comment (wesleyboar) [inline package-lock.json:12]: **Warning.** Do not commit the branch URL to this file. Re install/lock after merge of https://github.com/TACC/Core-Styles/pull/25.

File: package-lock.json [hunk 2]
```diff
@@ -3938,9 +3938,9 @@
       }
     },
     "@tacc/core-styles": {
-      "version": "git+https://git@github.com/TACC/Core-Styles.git#16c8873586785344ebd84b8360584c5a390bc499",
+      "version": "git+https://git@github.com/TACC/Core-Styles.git#115bcdc07bfbf8b97ad804f8e36ffc672234d330",
       "dev": true,
-      "from": "@tacc/core-styles@git+https://git@github.com/TACC/Core-Styles.git",
+      "from": "@tacc/core-styles@git+https://git@github.com/TACC/Core-Styles.git#bugfix/fp-1542-figure-image-margin",
```
Comment (wesleyboar) [inline package-lock.json:3943]: **Warning.** Do not commit the branch URL to this file. Re install/lock after merge of https://github.com/TACC/Core-Styles/pull/25.

File: package.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
     "npm": "^8.5.5"
   },
   "devDependencies": {
-    "@tacc/core-styles": "git+https://git@github.com/TACC/Core-Styles.git",
+    "@tacc/core-styles": "git+https://git@github.com/TACC/Core-Styles.git#bugfix/fp-1542-figure-image-margin",
```
Comment (wesleyboar) [inline package.json:12]: **Warning.** Do not commit the branch URL to this file. Re install/lock after merge of https://github.com/TACC/Core-Styles/pull/25.

File: taccsite_cms/templates/guides/data_transfer.html [hunk 4]
```diff
@@ -1,10 +1,11 @@
 {% extends "guide.html" %}
+{% load static %}
 
 {% block guide %}
   <style>
   /* To override 800px (from stylesheet) which is too big for these images */
   .s-document img {
-    max-width: 570px;
+    max-width: 670px;
```
Comment (wesleyboar) [inline taccsite_cms/templates/guides/data_transfer.html:8]: **Explanation.** Image text in Data Transfer's bottom-most images was easier to read if a bit larger.

**Review by wesleyboar — COMMENTED** [review]
Notes for reviewers.

File: package-lock.json [hunk 5]
```diff
@@ -64,7 +64,7 @@
     },
     "node_modules/@tacc/core-styles": {
       "version": "2.0.0",
-      "resolved": "git+https://git@github.com/TACC/Core-Styles.git#16c8873586785344ebd84b8360584c5a390bc499",
+      "resolved": "git+https://git@github.com/TACC/Core-Styles.git#ed1e102ffea902142845924c0c5357794e2117b6",
```
Comment (wesleyboar) [inline package-lock.json:67]: **Explanation.** To get fix from https://github.com/TACC/Core-Styles/pull/25.

**Review by wesleyboar — COMMENTED** [review]
Note for reviewer since updated core-styles commit.

**Review by taoteg — APPROVED** [review]
Would it be useful to scrub (or blur) the `A2CPS` from the paths in these images and replace it with `{PROJECT}`, so as not to confuse anyone referencing them?

- https://github.com/TACC/Core-CMS/pull/493/files#diff-4732bdc6c23df3069a4813d513406393610d216e79e26ff47ac29be80cc2dc66
- https://github.com/TACC/Core-CMS/pull/493/files#diff-3a5845b5aa10048e9dc314818891f3682af364686db06fbd4e14969a23710105

A similar concern with the right-side of this image: 

- [...e_cms/static/site_cms/img/guides/data_transfer.globus/step-2f-select-system.png](https://github.com/TACC/Core-CMS/pull/493/files#diff-7ef62a8eedcaff9aa80088950e528c19f23e5c803cd8a013330a18aed8bceb16)

Otherwise, LGTM as far as functional changes go. 
Approving as is, up to you on the above concerns.

It would be ideal if we had fresh clean screens from a dummy user actually setting everything up on current systems... but that's probably a bridge too far at this time. Maybe in the v3 iteration. =)

**Comment by wesleyboar** [comment]
Test URLs fixed:
- https://dev.cep.tacc.utexas.edu/guides/data-transfer-guide/
- https://dev.cep.tacc.utexas.edu/guides/globus-data-transfer-guide/

<sub>These will be good unless I need to deploy something else to dev.cep and am too lazy to create a branch with multiple features.</sub>

**Review by fnets — APPROVED** [review]
LGTM! Great job, this looks a lot cleaner.  Personally, I guess I'd prefer that the guide be a little more portal-agnostic like John said. But, I think that's outside the scope for this particular ticket.

**Comment by taoteg** [comment]
Preface: This all looks great and I don't know if this is in the scope of this PR, but... 

While comparing the Data Transfer Guide to the Globus Guide, I noticed that only the images in the Data Transfer Guide have indentation. Could we either A) make all guide images left-aligned or B) make all the guide images centered, with the outcome in either choice being consistent design across both the guides? 

I would vote for option B (tweak the Globus Guide layout to match the Data Transfer Guide layout by using centered images).

**Comment by wesleyboar** [comment]
It is out of scope for this bug fix, but there's also some typography bothering Hedda and I, so I want to add some miscellaneous updates in a follow-up ticket; I'll add your image alignment notes to it.

**Comment by wesleyboar** [comment]
Images have had project names blurred out (b16bb9f).[^1] I have not re-deployed.

Image alignment has been added to new ticket for Guide page inconsistencies: [FP-1676](https://jira.tacc.utexas.edu/browse/FP-1676). An existing ticket will add breadcrumbs: [FP-1452](https://jira.tacc.utexas.edu/browse/FP-1452).[^2]

[^1]: Replacing specific project name with `{PROJECT}` proved difficult to do attractively (match unknown font, fit in narrow gap).
[^2]: Non-crucial changes are low priority, because these Guides pages are supposed to be moved out to TUP documentation (or that's what I remember from a documentation meeting this year). But I do want to improve consistency, and needed to create a ticket for things I've noticed.

**Pull request merged**